Document PerformanceMark/PerformanceMeasure.detail (#21823)
mdn/content: files/en-us/web/api/performancemark/detail/index.md

---
title: PerformanceMark.detail
slug: Web/API/PerformanceMark/detail
page-type: web-api-instance-property
tags:
  - API
  - Property
  - Reference
  - Web Performance
browser-compat: api.PerformanceMark.detail
---

{{APIRef}}

> The read-only **`detail`** property returns arbitrary metadata that has been included in the mark upon construction (either when using {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).

Reviewer: ```suggestion
The read-only **`detail`** property returns arbitrary metadata that was included in the mark upon construction (either when using {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).
```

## Value

Returns the value it is set to (from `markOptions` of {{domxref("Performance.mark","performance.mark()")}} or the {{domxref("PerformanceMark.PerformanceMark","PerformanceMark()")}} constructor).

## Examples

> The following example shows the use of the `detail` property.

Reviewer: ```suggestion
The following example demonstrates the `detail` property.
```
